news/2022-08-25-changes-to-osu-taiko-star-rating-part-1.md

```diff
@@ -0,0 +1,68 @@
+
+Moving forward, osu!taiko star rating will be in a constant state of flux, particularly over the next year. This is generally desirable as we would want difficulty to accurately reflect the latest trends, but we understand and apologise if this may bring some unintended complications or frustrations, especially in the short term.  We are fully aware that this rework is not a complete solution in its current state.
+  
+## Release 
+- ☑ a bunch of checkboxes.
+  
+## Summary of changes
+
+### Stamina
+- Old stamina system assumes a two finger KDDK full-alt playstyle, with each hand being calculated separately.
+- New stamina system calculates the minimum stamina requirement theoretically needed for any pattern with 4 input keys (it assumes all 4 keys are independent with no handedness attached).
+- This does not accurately represent any single playstyle - this is desirable because we cannot feasibly detect playstyles. 
+- This "minimum" of sorts is therefore the fairest judgement of difficulty - any additional difficulty arising from specific patterning is attached to the playstyle itself.
+
+### Colour
+- Old colour system detected the frequency of colour changes and had penalties for specific repeated patterns.
+- New colour system evaluates all patterns using a compression algorithm, and adds those patterns to a dictionary.
+- The more information that is needed to describe a pattern, the harder it is.
+- Maps that have a larger dictionary (ie. use a large variety of patterns) will be evaluated as being harder.
+- The dictionary also has a "memory" of the patterns used. If a known pattern reappears later in the map, its difficulty will scale with a function of how long it last appeared in the map (ie. close repeated patterns are nerfed).
+
+### PP rework
+- osu!taiko PP has historically been calculated by summing the "Accuracy PP" and "Difficulty PP" components of a score. 
+
+#### Global Changes
+The global multiplier affects the final pp value of the map (ie. it is applied at the end of pp calculation).
+- Reduce the HD global multiplier (to increase the gap between EZHD and HD only scores)
+- Reduce the EZ global multiplier (to increase the gap between EZ and NM scores)
+- Addition of Effective Miss Count - this calculated as a ratio, with maps below 1000 combo receiving a multiplier to their misscount.
+
+#### Difficulty
+The following changes should result in impressive scores on harder mapsets rewarding more pp.
+- Reduce the miss penalty 
+- Increase the weight a score's accuracy has on Difficulty PP.
+- Removal of the NF multiplier if the score is "good enough" (this is significantly less than what is required for pass).
+
+#### Accuracy
+The following changes should result in high accuracy scores on harder mapsets rewarding more pp.
+- Significantly reduce GreatHitWindow's (300s) impact on Accuracy PP.
+- Introduction of an SR component to Accuracy PP, which replaces most of the scaling seen within GreatHitWindow.
```

Comment:
This kinda describes the technicalities of *what* was done, but the more important thing for this newspost is what the practical effects of this change are. Are there any? If so, what are they?

Reply:
Applied in b824956